src/Parser.elm

```diff
@@ -0,0 +1,45 @@
+module Parser exposing (..)
+
+{- Sample:
+   Compressed Twinkling Cobaltite    4397
+   Compressed Glowing Carnotite      2603
+   Glowing Cinnabar                  1918
+   Glistening Coesite                2740
+-}
+
+
+type alias OreEntry =
+    { name : String
+    , volume : Int
+    }
+
+
+parseOreInputLine : String -> Maybe OreEntry
+parseOreInputLine line =
+    let
+        parts =
+            String.words line
+                |> List.reverse
```

Comment:
In my Elm projects I often import `List` as `L` so that this becomes `L.reverse` -- this is purely a personal style preference. Just letting you know about this, so you can consider it.